### PR TITLE
Update dev-guide-wordpress.md

### PR DESCRIPTION
**Pull request by minaelee**
### What is changed, added or deleted? (Required)

- add selecting a database to Part 3
- add Part 5 for confirming connectivity
- clarify sentence with ambiguous pronoun antecedent

### Which TiDB version(s) do your changes apply to? (Required)
- [x] v7.5 (TiDB 7.5 versions)

### What is the related PR or file link(s)?

Issue #16036

### CODE REVIEW

**Review by dveeden — APPROVED** [review]

**Review by shizn — APPROVED** [review]

File: tidb-cloud/dev-guide-wordpress.md [hunk 1]
```diff
@@ -96,6 +97,12 @@ Configure the WordPress database connection to TiDB Serverless.
 
 2. Set up your WordPress site by visiting [localhost](http://localhost/) if you start the container on your local machine or `http://<your_instance_ip>` if the WordPress is running on a remote machine.
 
+### Step 5: Confirm the database connection
+
+1. Close the connection dialog for your cluster on the TiDB Cloud website, and open the **Chat2Query** terminal.
```
Comment (lilin90) [inline tidb-cloud/dev-guide-wordpress.md:102]: "console" is a more commonly used word when we refer to <https://tidbcloud.com/>.
```suggestion
1. Close the connection dialog for your cluster on the TiDB Cloud console, and open the **Chat2Query** terminal.
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**